packages/scoutgame-ui/src/components/projects/components/ProjectForm/ProjectForm.tsx

```diff
@@ -0,0 +1,166 @@
+                <TextField
+                  error={!!fieldState.error}
+                  helperText={fieldState.error?.message}
+                  multiline
+                  minRows={3}
+                  placeholder='Brief project summary'
+                  {...field}
+                />
+              )}
+            />
+          </Stack>
+          <Stack>
+            <FormLabel>Project Website</FormLabel>
+            <Controller
+              control={control}
+              name='website'
+              render={({ field, fieldState }) => (
+                <TextField
+                  error={!!fieldState.error}
+                  helperText={fieldState.error?.message}
+                  placeholder='https://example.com'
+                  {...field}
+                />
+              )}
+            />
+          </Stack>
+          <Stack>
+            <FormLabel>Repository</FormLabel>
+            <Controller
+              control={control}
+              name='github'
+              render={({ field, fieldState }) => (
+                <TextField
+                  error={!!fieldState.error}
+                  helperText={fieldState.error?.message}
+                  placeholder='https://github.com/example/project'
+                  {...field}
+                />
+              )}
+            />
+          </Stack>
+        </Stack>
+        <Divider />
+        <Stack gap={3}>
+          <Stack gap={1}>
+            <Typography color='secondary' variant='h6'>
+              Smart Contracts
+            </Typography>
+            <Typography lineHeight={2}>
+              Add your smart contracts to earn additional gems and to participate in partner rewards, such as the Taiko
+              AI challenge.
+              <br />
+              Sign a message with the wallet that deployed your contracts to prove ownership.
+            </Typography>
+            <ProjectSmartContractForm control={control} deployers={deployers} setDeployers={setDeployers} />
+          </Stack>
+        </Stack>
+        <Divider />
+        <Stack gap={3}>
+          <Stack gap={1}>
+            <Typography color='secondary' variant='h6'>
+              Team
+            </Typography>
+            <Typography>Split Project Based Rewards with your teammates.</Typography>
+            <ProjectTeamMemberForm control={control} />
+          </Stack>
+        </Stack>
+        <FormErrors errors={errors} />
+      </Stack>
+      <Stack
+        gap={2}
+        flexDirection='row'
+        justifyContent='flex-end'
+        position='sticky'
+        bottom={0}
+        p={2}
+        bgcolor='background.default'
+      >
+        <Button variant='outlined' color='primary' href={cancelLink} LinkComponent={Link}>
+          Cancel
+        </Button>
+        <LoadingButton
+          variant='contained'
+          color='primary'
+          onClick={onSave}
+          loading={isDisabled}
+          disabled={isDisabled || !isDirty}
```

Comment:
let me save the form no matter what.

Reply:
well we don't disable it if the form is invalid, but you need to add some input first to save the form